### DISCUSSION TITLE
Cache test get InvalidArgumentException at Laravel 9

### DISCUSSION THREAD

**Discussion by hms5232** [orchestral/testbench#346] [Q&A]
* Testbench Version: 7.11.0
* Laravel Version: 9.36.4
* PHP Version: 8.1.11
* Database Driver & Version: N/A

### Description:

Everything is fine at Laravel 8 (PHP 7.3\~8.1), but it doesn't work at Laravel 9 (PHP 8.0\~8.1):

```
   • Hms5232\LaravelTwinCache\Tests\TwinStoreTest > get twin
   InvalidArgumentException 

  Driver [twin] is not supported.

  at vendor/laravel/framework/src/Illuminate/Cache/CacheManager.php:109
    105▕ 
    106▕             if (method_exists($this, $driverMethod)) {
    107▕                 return $this->{$driverMethod}($config);
    108▕             } else {
  ➜ 109▕                 throw new InvalidArgumentException("Driver [{$config['driver']}] is not supported.");
    110▕             }
    111▕         }
    112▕     }
    113▕
```
Seems the cache store `twin` doesn't loaded? However, Laravel 9 doesn't change any document about this part, and PHP 8.1/Laravel 8 is work, so may not caused by PHP version.

Thank you in advance for your helping.

### Steps To Reproduce:

1. Download https://github.com/hms5232/laravel-twin-cache/tree/laravel-9
2. composer install with PHP 8.1 (make sure Laravel version is `^9.0`)
3. run `composer test`

**Comment by crynobone** [comment]
You have to setup the configuration.

**Reply by hms5232** [reply]
> You have to setup the configuration.

Yes, I have a file setup everything: [`tests/TestCase.php`](https://github.com/hms5232/laravel-twin-cache/blob/laravel-9/tests/TestCase.php), include enviroment ([line 39](https://github.com/hms5232/laravel-twin-cache/blob/laravel-9/tests/TestCase.php#L39)) package provider ([line 16](https://github.com/hms5232/laravel-twin-cache/blob/laravel-9/tests/TestCase.php#L16)) and discovery ([line 28](https://github.com/hms5232/laravel-twin-cache/blob/laravel-9/tests/TestCase.php#L28)):

```php
class TestCase extends \Orchestra\Testbench\TestCase
{
    /**
     * Get package providers.
     *
     * @param  \Illuminate\Foundation\Application  $app
     *
     * @return array
     */
    protected function getPackageProviders($app)
    {
        return [
            'Hms5232\LaravelTwinCache\TwinCacheServiceProvider',
        ];
    }

    /**
     * Ignore package discovery from.
     *
     * @return array
     */
    public function ignorePackageDiscoveriesFrom()
    {
        return ['hms522/laravel-twin-cache'];
    }

    /**
     * Define environment setup.
     *
     * @param  \Illuminate\Foundation\Application  $app
     * @return void
     */
    protected function defineEnvironment($app)
    {
        // change cache config
        $app['config']->set('cache.default', 'twin');
        $app['config']->set('cache.prefix', 'twin');
        $app['config']->set('cache.stores', [
            'twin' => [
                'driver' => 'twin',
                'older' => 'array',
                'younger' => 'file',
                'sync_ttl' => 120,
            ],

            // Laravel bulit-in
            'array' => [
                'driver' => 'array',
                'serialize' => false,
            ],
            'file' => [
                'driver' => 'file',
                'path' => storage_path('framework/cache/data'),
            ],
        ]);
    }

    // other config
}
```

Anything i miss? or Laravel 9 has changed?
